packages/interactivity/src/directives.js

```diff
@@ -177,9 +177,11 @@ export default () => {
 							: name;
 
 					useInit( () => {
-						// This seems necessary because Preact doesn't change the class
-						// names on the hydration, so we have to do it manually. It doesn't
-						// need deps because it only needs to do it the first time.
+						/*
+						 * This seems necessary because Preact doesn't change the class
+						 * names on the hydration, so we have to do it manually. It doesn't
```

Comment:
WPCS: These and all other uses of "we" will need to be removed for inclusion in Core.

Reply:
Funny thing, this code is already in Core 😄

I'll review all the JS comments anyway for WP 6.5. Thanks, Dennis.